src/main/kotlin/org/wfanet/measurement/common/testing/CloseableResource.kt

```diff
@@ -23,19 +23,20 @@ import org.junit.runners.model.Statement
  * evaluation.
  *
  * This can replace [ExternalResource][org.junit.rules.ExternalResource], which does not guarantee
- * that [after][org.junit.rules.ExternalResource.after] is invoked if
- * [before][org.junit.rules.ExternalResource.before] throws an exception.
+ * that [after][org.junit.rules.ExternalResource.after] is invoked if [before]
+ * [org.junit.rules.ExternalResource.before] throws an exception.
  */
 open class CloseableResource<T : AutoCloseable>(private val createResource: () -> T) : TestRule {
   protected lateinit var resource: T
     private set
 
-  override fun apply(base: Statement, description: Description) = object : Statement() {
-    override fun evaluate() {
-      check(!::resource.isInitialized)
+  override fun apply(base: Statement, description: Description) =
```

Comment:
Convert to block body?

Reply:
Will do. It looks like the rule ktfmt uses for wrapping these depends on whether the expression needs parens for parameters. In this case, `Statement` is an abstract class such that the expression needs parens so it gets gets wrapped. In comparison, `TestRule` is an interface so the expression doesn't need parens.

Reply:
Done.